composer.json

```diff
@@ -53,6 +57,12 @@
             }
         }
     },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://nova.laravel.com"
+        }
+    ],
```

Comment:
It seems like this should also be added to every package.

Reply:
Yes.  It was part of the delta I flagged for every repo.